proxy/proxy.go

```diff
@@ -269,23 +262,27 @@ func modifyProxiedResponse(config *config.Config, cache cache.Cache, cacheKey st
 			if uplinkResponse.Data.RouterEntitlements.Entitlement != nil {
 				jwt = uplinkResponse.Data.RouterEntitlements.Entitlement.Jwt
 			}
-
+			expiration, err := time.Parse(time.RFC3339, uplinkResponse.Data.RouterEntitlements.ID)
```

Comment:
```suggestion
                         // TODO: Add user docs on the time format supported 
			expiration, err := time.Parse(time.RFC3339, uplinkResponse.Data.RouterEntitlements.ID)
```

Reply:
Committing, but for posterity the time format is ISO8601 which can be parsed using RFC3339. :)